routers/repo/issue.go

```diff
@@ -605,6 +605,29 @@ func commentTag(repo *models.Repository, poster *models.User, issue *models.Issu
 	return models.CommentTagNone, nil
 }
 
+func getBranchData(ctx *context.Context, issue *models.Issue) {
+	ctx.Data["BaseBranch"] = nil
+	ctx.Data["HeadBranch"] = nil
+	ctx.Data["HeadUserName"] = nil
+	ctx.Data["BaseName"] = ctx.Repo.Repository.OwnerName
+	if issue.IsPull {
+		pull := issue.PullRequest
+		ctx.Data["BaseBranch"] = pull.BaseBranch
+		ctx.Data["HeadBranch"] = pull.HeadBranch
+		headUserName := pull.MustHeadUserName()
+		ctx.Data["HeadUserName"] = headUserName
+
+		headUser, err := models.GetUserByName(headUserName)
+		if err != nil {
+			ctx.ServerError("getBranchData", err)
+			return
+		}
+		if headUser.ID == ctx.Repo.Owner.ID {
+			ctx.Data["BaseName"] = headUser.Name
```

Comment:
Why override line 612?

Reply:
I just double checked. Might have been needed when I started the PR months ago.
Doesn't seem to be needed anymore - I removed the lines.